go.mod

```diff
@@ -20,38 +20,4 @@ require (
 	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.0
 )
 
-require (
```

Comment:
I see now some 'go mod tidy' issues in tests. I think this is an incorrect deletion; try running tidy with go 1.17.

Reply:
fixed